### PR TITLE
Add Github Actions support

### PR DESCRIPTION
**Pull request by gmhabchi**
Add support for Github Actions and based config off of the Travis CI config 👍 

ref: #1346

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.16]
```
Comment (moltar) [inline .github/workflows/build.yml]: if using matrix, then worth testing on other versions too. can also add a mix of OSs:

```
jobs:
  test:
    name: Test on node ${{ matrix.node }} and ${{ matrix.os }}

    runs-on: ${{ matrix.os }}

    strategy:
      matrix:
        node:
          - 8
          - 10
          - 12
        os:
          - ubuntu-latest
          # - windows-latest
          - macOS-latest
```

Comment (gmhabchi) [reply .github/workflows/build.yml]: Not a problem! 👍

Comment (gmhabchi) [reply .github/workflows/build.yml]: I can add other yml files for the different OS since it isn't just npm commands.

Comment (sobolevn) [reply .github/workflows/build.yml]: Other versions would conflict with the engine we define in `package.json`
And the best practice for apps is to have one specific version.

Comment (gmhabchi) [reply .github/workflows/build.yml]: Okay, I'm pretty sure this is back to what I had at the start. But are you guys happy with this now?

**Comment by sobolevn** [comment]
Let's test it!

**Comment by sobolevn** [comment]
Link: https://github.com/wemake-services/wemake-vue-template/commit/8012e694de21cc190dd1cce4e48f004a929bf41f/checks?check_suite_id=284066177

**Comment by sobolevn** [comment]
@gmhabchi looks like we have missed this step: https://github.com/wemake-services/wemake-vue-template/blob/master/.travis.yml#L33

Would please add it?

**Comment by gmhabchi** [comment]
Looks like Github Actions doesn't support env and/or .env the same way that Travis CI does. The only way I can think to fix this is to change the scripts or just move those tests into Github Actions and run directly there

**Comment by sobolevn** [comment]
Ok, let's try 👍

File: .github/workflows/build.yml (hunk 1 above)
Comment (moltar) [reply .github/workflows/build.yml]: > And the best practice for apps is to have one specific version.

Right, but I don't think it's the best practice to dictate which version of Node the developer / production should run in. Sometimes you cannot control which Node version is installed. That is why it should be tested on all major Node versions.

**Pull request merged**